pkg/users/account-create-dialog.js

```diff
@@ -284,20 +338,33 @@
 
         errs.user_name = validate_username(user_name, accounts);
 
+        const promises = [
+            password_quality(password, force_weak)
+                    .catch(ex => {
+                        errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230324-000238-1c01f86d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)